src/clr-angular/forms/common/providers/ng-control.service.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import {Injectable} from "@angular/core";
+import {NgControl} from "@angular/forms";
+import {Observable, Subject} from "rxjs";
+
+@Injectable()
+export class NgControlService {
```

Comment:
Now that I see it implement, this is literally a `Subject`, or `BehaviorSubject` if you add the getter again.
You don't need to change it, and I get that we might want to add properties or methods to it in the future, I just thought it was worth pointing out this is a bit of a silly wrapper.

Reply:
Yeah once I detangled it its pretty light. I do like the idea of having the method do the emit instead of exposing it.